### ISSUE TITLE
Set up lottery reviewing

### ISSUE THREAD

**Issue by illright** [aabounegm/cast#5]
Use [this GitHub Action](https://github.com/marketplace/actions/reviewer-lottery). Create an escape route to skip code review when a feature was pair-programmed.

**Acceptance criteria**:
- every subsequent PR should have 1 person from the team assigned for review randomly **after the CI checks pass**
- PRs with a `pair-programmed` label should not be affected.

**Comment by aabounegm** [comment]
Opening again because it did not work (on #33). On a quick glance, it seems that `tags-ignore` is only supported for `push` not `pull_request`, and it refers to git tags and not PR labels, so it seems like some manual code will need to be written.

Also, about the 1 approval that is required anyway to merge, is the other pair programmer required to do it? If so, shouldn't he be the one whose review is requested?

**Comment by alkaitagi** [comment]
> Opening again because it did not work (on #33).

Please have a look at #34.

**Comment by illright** [comment]
> Also, about the 1 approval that is required anyway to merge, is the other pair programmer required to do it? If so, shouldn't he be the one whose review is requested?

Yeah, in cases of pair programming I imagine that the author makes a PR and the partner immediately approves it
